crates/swc_ecma_transforms_optimization/src/simplify/expr/tests.rs

```diff
@@ -843,6 +843,12 @@ fn test_fold_arithmetic_infinity() {
     fold("x = Infinity ** -2", "x = 0");
 }
 
+#[test]
```

Comment:
I think this is not a good place to add a test. You can add a test in https://github.com/swc-project/swc/tree/2f22fee353e7e6be013d5ee9335bd558f20bd72b/crates/swc/tests/fixture/issues-9xxx and run `UPDATE=1 cargo test --test projects` from `./crates/swc` instead.